### DISCUSSION TITLE
Find Sharepoint App Only Service Principal Permissions

### DISCUSSION THREAD

**Discussion by stvpwrs** [pnp/powershell#142] [Q&A] [answered]
Hello,

Is there a way to view all of the currently assigned permissions to a SharePoint App Only Service Principal? I can see them listed, but I can't seem to find anywhere, in AD or SharePoint that then lists if it has tenant or site access etc. 

Is there a way to get that?

Steve

**Comment by erwinvanhunen** [comment, marked as answer]
As far as I know this not exposed through any of the APIs.
